includes/Core/Admin/Screens.php

```diff
@@ -288,13 +288,16 @@ private function get_screens() {
 						}
 					},
 					'render_callback'  => function( Context $context ) {
+						$setup_slug = $context->input()->filter( INPUT_GET, 'slug', FILTER_SANITIZE_STRING );
+						$reauth = $context->input()->filter( INPUT_GET, 'reAuth', FILTER_SANITIZE_STRING );
 						if ( $context->input()->filter( INPUT_GET, 'permaLink' ) ) {
 							?>
 							<div id="js-googlesitekit-dashboard-details" class="googlesitekit-page"></div>
 							<?php
 						} else {
+							$setup_module_slug = strlen( $setup_slug ) && strlen( $reauth ) ? $setup_slug : '';
```

Comment:
Given the types, I think we can simplify this like so:
```suggestion
							$setup_module_slug = ( $setup_slug && $reauth ) ? $setup_slug : '';
```

Reply:
https://github.com/google/site-kit-wp/pull/3805/commits/861db7225e67b969e69a3afd7f851016096a7859